newlib/libc/search/qsort.c

```diff
@@ -187,7 +193,7 @@ qsort (void *a,
 	char *pa, *pb, *pc, *pd, *pl, *pm, *pn;
 	size_t d, r;
 	int cmp_result;
-	int swaptype, swap_cnt;
+	int swaptype_big_primitive_type, swap_cnt;
```

Comment:
Why does this need renaming?

Reply:
The macros now use that variable name (`swaptype_big_primitive_type`) which is more indicative?

Reply:
Not really. It's only called that in FreeBSD because there's also a `swaptype_int`. This just unnecessarily adds to the diff for purely cosmetic reasons (that I don't think are even good).